app/services/CalculationService.scala

```diff
@@ -35,6 +36,9 @@ trait CalculationService {
   val higherRate = higherRatePercentage / 100.toDouble
   val basicRateBand = 32000
   val startOfTax = "2015-04-06"
+  val startOfTaxDateTime = DateTime.parse("2015-04-06")
+  val disposalDateTimePRR = DateTime.parse("2016-10-06")
```

Comment:
I don't think this needs a `disposalDateTimePRR` declaration for the 6/10/2016. Instead, this is the `startOfTaxDateTime + 18 Months`. 

So where you're currently referring to `disposalDateTimePRR` you could replace this with `startOfTaxDateTime.addMonths(eightenMonths)`